index.html

```diff
@@ -0,0 +1,107 @@
+<!doctype html>
+<html lang="en" ng-app="memberApp">
```

Comment:
Probably better to use angular.bootstrap vs ng-app gives more flexibility when you get onto more complex coding